client/src/components/saveModal.jsx

```diff
@@ -0,0 +1,310 @@
+`
+const ModalClose = styled.span`
+  position: relative;
+  left: 50%;
+  color: #666; 
+  &:hover{
+    color: #BCBCBC;
+    cursor: pointer;
+  }
+`
+ModalClose.displayName = "ModalClose"
+
+const ShareButtonContainer = styled.div`
+  margin: 20px 0px;
+`
+const SignUpWithFacebook = styled.div`
+  text-align: center;
+  background-color: #3B5998;
+  box-shadow: 0 1px 1px rgba(0,0,0,0.3);
+  color: white;
+  width: 100%;
+  height: 100%;
+  border-radius: 5px;
+  vertical-align: middle;
+  padding: 5px 0px;
+  border-color: #263961;
+  border: 1px solid rgb(38, 57, 97);
+  font-weight: bold;
+  font-size: .9rem;
+  margin-bottom: 10px;
+  &:hover{
+    cursor: pointer;
+    background-color: #4669B5;
+  }
+`
+  
+const ShareIcon = styled.i`
+  margin-right: 15px;
+`
+
+const SignUpWithGoogle = styled.div`
+  box-shadow: 0 1px 1px rgba(0,0,0,0.3);
+  text-align: center;
+  background-color: white;
+  color: rgb(102,102,102);
+  width: 100%;
+  height: 100%;
+  border-radius: 5px;
+  vertical-align: middle;
+  padding: 5px 0px;
+  border-color: #263961;
+  border: 1px solid rgb(217, 217, 217);
+  font-weight: bold;
+  font-size: .9rem;
+  &:hover{
+    cursor: pointer;
+    background-color: #f5f5f5;;
+}
+`
+const OrFieldSet = styled.fieldset`
+  border-top: 1px solid #ccc;
+  border-bottom: 1px solid transparent;
+  text-align: center;
+  margin: 15px 0 0 0;
+  border-right: none; 
+  border-left: none; 
+  `
+const OrLegend = styled.legend`
+  font-size: .85em;
+  padding: 0 18px;
+  color: #666;
+`
+
+const SignUpForm = styled.form`
+  text-align: left;
+`
+
+const SignUpLabel = styled.label`
+  color: black;
+  font-weight: 600;
+  display: inline-block;
+  margin-bottom: 3px;
+  font-size: 14px;
+  margin-right: 5px;
+`
+
+const SignUpInput = styled.input`
+  height: 30px;
+  width: 100%;
+  margin-bottom: 13px;
+  border: 1px solid rgb(153, 153, 153);
+  border-radius: 3px;
+  padding: 5px;
+  box-sizing: border-box;  
+`
+const ZipCodeInput = styled.input`
+  height: 30px;
+  width: 100%;
+  margin-bottom: 10px;
+  border: 1px solid rgb(153, 153, 153);
+  border-radius: 3px;
+  padding: 5px;
+  box-sizing: border-box;  
+`
+
+const FirstNameInput = styled.input`
+  height: 18px;
+  width: 50%;
+  display: inline;
+  border: 1px solid rgb(153, 153, 153);
+  border-radius: 3px;
+  padding: 5px;
+  margin-right: 10px;
+`
+const LastNameInput = styled.input`
+  height: 18px;
+  width: 50%;
+  display: inline;
+  border: 1px solid rgb(153, 153, 153);
+  border-radius: 3px;
+  padding: 5px;
+`
+
+const SubTitle = styled.p`
+  display: inline; 
+  color: #999999; 
+  font-size: .75rem;
+  font-weight: normal;
+  text-align: right;
+`
+const ShareModalButton = styled.div`
+  width: 100%;
+  box-sizing: border-box;
+  text-align: center; 
+  border-radius: 3px;
+  box-shadow: rgba(0, 0, 0, 0.3) 0px 1px 1px 0px;
+  border-style: solid 1px;
+  padding: 8px;
+  margin-right: 12px;
+  margin-bottom: 12px;
+  background-color: #DF1201;
```

Comment:
again, best practices, but you seem to be jumping between using hex and rgb. 
Might count as a chore this far in, but should be consistent

Reply:
yep haha i was copying colors from a couple of different sources - chrome dev tools uses rgb for the most part, but i have a color picking chrome extension that gives me the color in hex. I'll make the changes as I see them!

Reply:
I figured that was the source of the differences. thanks, and definitely something to keep in mind moving forward